src/PerfCounters.cc

```diff
@@ -121,6 +122,7 @@ struct PmuConfig {
 // See Intel 64 and IA32 Architectures Performance Monitoring Events.
 // See check_events from libpfm4.
 static const PmuConfig pmu_configs[] = {
+  { IntelIcelake, "Intel Icelake", 0x5111c4, 0, 0x5301cb, 0, 100, PMU_TICKS_RCB },
```

Comment:
Can we replace 0x5301cb with 0 here? I don't think that counter actually exists on Ice Lake.

Reply:
Done.  Zero seems to make no difference.

Reply:
Yeah that entry in the table isn't actually used now.